sympy/physics/units/dimensions.py

```diff
@@ -149,36 +153,43 @@ def __add__(self, other):
         return self
 
     def __sub__(self, other):
-        # there is no notion of ordering (or magnitude) among dimension,
-        # subtraction is equivalent to addition when the operation is legal
+        """There is no notion of ordering (or magnitude) among dimension.
+
+        Subtraction is equivalent to addition when the operation is legal.
+        """
         return self + other
 
     def __pow__(self, other):
+        """Forward exponent to dimension name."""
         return self._eval_power(other)
 
     def _eval_power(self, other):
         other = sympify(other)
         return Dimension(self.name**other)
 
     def __mul__(self, other):
+        """Multiplication works only between two dimensions."""
         if not isinstance(other, Dimension):
             return self
 
-        return Dimension(self.name*other.name)
+        return Dimension(self.name * other.name)
 
     def __div__(self, other):
+        """Division works only between two dimensions."""
```

Comment:
There is no need to add docstrings to dunder methods, in general. We don't require that and you'll find that the large majority do not have them.